src/migration/1730903564613-UserTopReader.ts

```diff
@@ -0,0 +1,17 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class UserTopReader1730903564613 implements MigrationInterface {
+  name = 'UserTopReader1730903564613'
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`CREATE INDEX IF NOT EXISTS "IDX_post_keyword_status" ON "post_keyword" ("status")`);
+    await queryRunner.query(`CREATE UNIQUE INDEX IF NOT EXISTS "IDX_user_top_reader_userId_issuedAt_keywordValue" ON "user_top_reader" ("userId", "issuedAt", "keywordValue") `);
```

Comment:
This intended, don't see it defined anywhere?

Reply:
Forgot to add it. Done now.